omigami/utilities/spectrum_df_helper.py

```diff
@@ -0,0 +1,67 @@
+from typing import List, Optional, Tuple
+
+import pandas as pd
+from matchms import Spectrum
+from pandas.tests.groupby.test_value_counts import df
+
+
+class SpectrumDataFrameHelper:
+    def __init__(self):
+        pass
+
+    @staticmethod
+    def scale(spectra_df: pd.DataFrame, min=0, max=1, num_peaks=5000) -> pd.DataFrame:
+        """
+        Scale the Intensity column according to min and max values. By default, normalizes to 0 and 1.
+        """
+        df = spectra_df.copy()
+
+        min_intensity, max_intensity = df.Intensity.min(), df.Intensity.max()
+        df["Intensity"] = (df.Intensity - min_intensity) / (
+            max_intensity - min_intensity
+        ) * (max - min) + min
```

Comment:
you didn't want to use scikit-learn's min max scaler?

Reply:
and if it is min max scaling why do we have `* (max - min) + min` at the very end?

Reply:
Yes, I didn't want to introduce scikit as a dependency just to use a simple function. This last part scales to the min and max passed in as parameters. If not passed, it defaults to 0 and 1 (which would pretty much be a multiplication by 1 in that case)